demos/ASP.Net Core/ITOps.ViewModelComposition.Gateway/ComposableRouteHandler.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Primitives;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+using System.Threading.Tasks;
+
+namespace ITOps.ViewModelComposition.Gateway
+{
+    class ComposableRouteHandler
+    {
+        public static async Task HandleGetRequest(HttpContext context)
+        {
+            var result = await CompositionHandler.HandleGetRequest(context);
+            if (result.StatusCode == StatusCodes.Status200OK)
+            {
+                string json = JsonConvert.SerializeObject(result.ViewModel, GetSettings(context));
+                context.Response.ContentType = "application/json; charset=utf-8";
```

Comment:
![image](https://user-images.githubusercontent.com/677704/28465403-84b0e05e-6e20-11e7-8256-141973c86c53.png)

OK, I think for the sample it's good enough.

Perhaps we should add a comment here to avoid an attendee asking the same question? Something like:

```c#
// For the purposes of the sample, we're not respecting the HTTP Accept header and assuming that a JSON response is OK.
```